ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml

```diff
@@ -7743,3 +7743,30 @@ CHCPC2X:
   - chilled_water_circulation_pump_run_status_2
   - chilled_water_circulation_pump_run_command_1
   - chilled_water_circulation_pump_run_command_2
+
+ESPM2X:
+  guid: "3d2d7498-e741-4690-aab6-f2e14e11cfeb"
+  description: "Dual electrostatic precipitator monitoring."
+  is_abstract: true
+  opt_uses:
+  - failed_electrostatic_precipitator_alarm_1
+  - failed_electrostatic_precipitator_alarm_2
+  uses:
+  - electrostatic_precipitator_run_command_1
+  - electrostatic_precipitator_run_command_2
+  - electrostatic_precipitator_run_status_1
+  - electrostatic_precipitator_run_status_2
+  implements:
+  - MONITORING
+
+DTPM:
+  guid: "898b43c5-c051-4fbf-8366-4c1fefda40b6"
+  description: "Detergent pump monitoring."
+  is_abstract: true
+  opt_uses:
+  - low_detergent_level_alarm
```

Comment:
revise to be `low_detergent_alarm` (level is somewhat redundant, and this aligns with the internal standards)